sagemaker-python-sdk/mxnet_gluon_mnist/mxnet_mnist_with_gluon_1.3.ipynb

```diff
@@ -0,0 +1,1139 @@
+{
```

Comment:
if the only difference here is that it's using MXNet 1.3 instead of 1.2, we should just update the old notebook rather than making a new one.